tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest

```diff
@@ -0,0 +1,5 @@
+RUN: %strictdoc export --formats=excel --fields=uid,owner --output-dir=Output "%S/input.sdoc"
```

Comment:
I have checked the expected Excel file manually, and it looks like you are not actually testing most of the things that you have in the `input.sdoc`. Could you remove the `--fields` argument, regenerate the Excel file, make sure that it looks like as expected, and finally force-push your change?

Reply:
Added some new Export Excel ITests. Testing showed that the specials "Parent" and "Comments" needed some work.

Reply:
The new PR failed the ITest "07_custom_grammar_fields_to_export_fields_all_special" in the "Periodic Integration Test" 
Seems that pybtex somewhre needs the Six package? Not sure where to include/fix this.

Reply:
That part of the integration test has to do with a rather tricky setup of PyInstaller. I am hoping to take a look into this still today.